addons/realisticnames/CfgVehicles.hpp

```diff
@@ -363,6 +392,11 @@ class CfgVehicles {
         displayName = CSTRING(Plane_Fighter_03_AA_Name);
     };
 
+    class Plane_Fighter_03_dynamicLoadout_base_F;
+    class I_Plane_Fighter_03_dynamicLoadout_F: Plane_Fighter_03_dynamicLoadout_base_F {
+        displayName = CSTRING(Plane_Fighter_03_Name);
+    };
+
     // uavs
     class UAV_02_base_F;
```

Comment:
I was wondering while looking through the configs here, why are we not just adding the names to the base classes where applicable? This one for example, they all inherit from it anyway.